src/utils/__tests__/Results.utils.test.js

```diff
@@ -0,0 +1,80 @@
+import {
+  downloadExperimentRunResult,
+  formatCompareResultDate,
+  formatResultsParameters,
+  formatTensorValues,
+  transformResults,
+} from 'utils/Results.utils';
+
+describe('Results utils', () => {
+  it('should transform results with an operator ID', () => {
+    const results = [{ type: 'plot', uuid: 'plot_uuid', plotUrl: 'plot_url' }];
+    const transformedResults = transformResults('OPERATOR', results);
+    expect(transformedResults).toEqual([
+      { type: 'plot', uuid: 'plot_OPERATOR_0', plotUrl: results[0] },
+    ]);
+  });
+
+  it('should format the date of a result', () => {
+    const date = new Date(1629477399451);
+    expect(formatCompareResultDate(date)).toBe('20 de agosto , 13:36');
+  });
```

Comment:
@LuanEdCosta este teste falhou porque usou o idioma do amb. de CI (inglês):
![image](https://user-images.githubusercontent.com/1791381/130652748-757fb3fe-bdb4-4059-9aa9-bba6240b2d09.png)

Como você acha a melhor forma de lidar com isso? :x
Rodar o teste apenas se o idioma for pt?
Alterar a implementação do format pra sempre retornar em pt-BR?

Reply:
Ah acho que eu consigo pegar o locale do ambiente para formatar a data, se eu não conseguir vou ver se faço o teste passar de alguma forma. Pode ser?

Reply:
Fiz o teste com o locale en-US e funcionou, mesmo não sendo o mais adequado, mas acredito que o comportamento está sendo testado